pkg/platform/common/vmlayer/gpu.go

```diff
@@ -330,13 +332,21 @@ var NvidiaGPUOperatorApp = edgeproto.App{
 `,
 		},
 	},
+	// TODO: what resources does the NVidia operator need?
+	// should those resources be tracked by the controller?
+	KubernetesResources: &edgeproto.KubernetesResources{
+		CpuPool: &edgeproto.NodePoolResources{
+			TotalVcpus:  *edgeproto.NewUdec64(0, 200*edgeproto.DecMillis),
+			TotalMemory: 200,
+		},
+	},
```

Comment:
Another tracking issue maybe for this?

Reply:
Yeah, let me open a general issue around tracking sidecar/OS resources.